lib/timers.js

```diff
@@ -111,16 +111,20 @@ function enroll(item, msecs) {
   item._idleTimeout = msecs;
 }
 
+function isValidCallback(callback) {
```

Comment:
Ok. Or we could use arrow function which no stack call
```suggestion
const isValidCallback = (callback) => {
```

Reply:
Done 👍

Reply:
@Himself65 Making this an arrow function will still let it show up in the call stack just like a regular function. Your suggestion does not affect that behavior in any way. 

@yashLadha We have a `hideStackFrames()` helper that actually does what @Himself65 seems to be concerned about. Feel free to use that.

Reply:
Oh, I was going to say `context`, not `stack`, my mistake :X

Reply:
@Himself65 may I ask if your request changes status is still valid here?

Reply:
sorry for I forgot this PR